test-case-reporting/app.ts

```diff
@@ -142,12 +143,19 @@ app.post('/report', jsonParser, async (req, res) => {
 });
 
 app.get('/_cron/compute-stats', async (req, res) => {
+  const testCaseStats = new TestCaseStats(db);
```

Comment:
This can be created up top outside of the request handler, just like the `TestResultRecord`

Reply:
Done. Why do we want to create it up top, if it's only used by this endpoint?

Reply:
Because then it's only created once and shared between requests, which is mildly preferable